### DISCUSSION TITLE
Move `cpanfile` to new `perl` group in Supported Managers table?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#23805] [Suggest an Idea]
### Tell us more.

@secustor The Supported Managers table puts the `cpanfile` in the `No Category`. Maybe the `cpanfile` manager should go in a new `perl` category? So something like this:

| Group | Category ID | Managers |
| ----- | ----------- | -------- |
| Perl  | `perl`      | `cpanfile` |

The `cpanfile` should only go in the `perl` category if _only Perl_ uses that `cpanfile`. If other programming languages use `cpanfile` it should probably stay in the `No category`. 😄 

Related PR:

- #23503

**Comment by secustor** [comment]
I'm not familiar with cpanfile, tough if a `Categories` variable is added to the manager the manager will be moved in the correct categories. 

Based on a quick Google search CPAN only supports Perl.

**Reply by viceice** [reply]
yes, cpan is the perl package manager. so it makes sense for a perl category there.

**Reply by secustor** [reply]
Implemented in https://github.com/renovatebot/renovate/pull/23808
